src/modules/src/controller_lee_payload.c

```diff
@@ -196,6 +205,9 @@ static controllerLeePayload_t g_self = {
   .KR = {0.008, 0.008, 0.01},
   .Komega = {0.0013, 0.0013, 0.002},
   .KI = {0.02, 0.02, 0.05},
+  // desired orientation and omega for payload
+  .qp_des = {0, 0, 0, 1},
+  .wp_des = {0, 0, 0},
```

Comment:
this should use the data from setpoint (attitudeQuaternion and omega).

Reply:
done